.rubocop.yml

```diff
@@ -0,0 +1,409 @@
+# The behavior of RuboCop can be controlled via the .rubocop.yml
+# configuration file. It makes it possible to enable/disable
+# certain cops (checks) and to alter their behavior if they accept
+# any parameters. The file can be placed either in your home
+# directory or in some project directory.
+#
+# RuboCop will start looking for the configuration file in the directory
+# where the inspected file is and continue its way up to the root directory.
+#
+# See https://docs.rubocop.org/rubocop/configuration
+
+AllCops:
+  NewCops: disable
+  TargetRubyVersion: 2.3
```

Comment:
@dkniffin 2.3? That old?

Reply:
Oh, whoops. I just copied what I had here, but without the `>=` https://github.com/Quimbee/importmap-package-manager/blob/main/importmap-package-manager.gemspec#L11 🤦‍♂️ I'll update this

Reply:
Hmm, actually those two values should match, so I did that right. But 2.3 might be older than we want to support. It looks like [importmap-rails supports 2.7+](https://github.com/rails/importmap-rails/blob/main/importmap-rails.gemspec#L17). I think I'll go with that same value, unless you can think of a reason not to.

Reply:
PR for that here: https://github.com/Quimbee/importmap-package-manager/pull/3

Reply:
I think following same supported version as `importmap-rails` is a great idea.